Cargo.lock

```diff
@@ -1,1863 +1,1893 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
I'll be honest...I don't know enough Rust to know if it's fine that this changed as much as it did.